packages/build/src/plugins_core/functions/index.js

```diff
@@ -20,26 +20,31 @@ const { getZipError } = require('./error')
 // The function configuration keys returned by @netlify/config are not an exact
 // match to the properties that @netlify/zip-it-and-ship-it expects. We do that
 // translation here.
-const normalizeFunctionConfig = (functionConfig = {}) => ({
-  externalNodeModules: functionConfig.external_node_modules,
-  ignoredNodeModules: functionConfig.ignored_node_modules,
+const normalizeFunctionConfig = ({ buildDir, config = {} }) => ({
+  externalNodeModules: config.external_node_modules,
+  includedFiles: config.included_files,
+  includedFilesBasePath: buildDir,
+  ignoredNodeModules: config.ignored_node_modules,
 
   // When the user selects esbuild as the Node bundler, we still want to use
   // the legacy ZISI bundler as a fallback. Rather than asking the user to
   // make this decision, we abstract that complexity away by injecting the
   // fallback behavior ourselves. We do this by transforming the value
   // `esbuild` into `esbuild_zisi`, which zip-it-and-ship-it understands.
-  nodeBundler: functionConfig.node_bundler === 'esbuild' ? 'esbuild_zisi' : functionConfig.node_bundler,
+  nodeBundler: config.node_bundler === 'esbuild' ? 'esbuild_zisi' : config.node_bundler,
```

Comment:
[sand] We might want to keep naming it `functionConfig` to differentiate it from the top-level `config`, what do you think?

Reply:
Since the function is named `normalizeFunctionConfig`, I felt it was clear enough that the argument referred to the function config and opted for `config` for conciseness. But I'm happy to revert to `functionConfig` if you think that's clearer.

Reply:
Any variable name could work, but I do think it would be useful to use different variable names for different concepts. For example, if we use a consistent variable name for the object at `config.functions`, it allows us to `grep` this locally. 
This is not a blocker though, so please feel free to disregard, `config` works too.

Reply:
Sure, happy to change it. Done in https://github.com/netlify/build/pull/2681/commits/b3487b72453efe08cb86420b1ae5382b865a86ea.